components/camview.py

```diff
@@ -70,32 +64,41 @@ def move(self, event):
 
     def handle_wheel(self, event):
         """(Currently unused) Reacts to mouse wheel changes."""
-        return
+        # return
 
         delta = event.GetWheelRotation()
+        print(delta)
```

Comment:
If all print statements were just for checking values, please remove all of them.

Reply:
Oops, removed all of them